internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json

```diff
@@ -8442,7 +8442,6 @@
         {
           "name": "Oslo",
           "types": [
-            "AdministrativeArea3",
```

Comment:
@n-h-diaz is there some place import group change?

Reply:
Related to cl/638945277 : Revert NUTS to AA mapping.